### ISSUE TITLE
Reg\Register an Operation\Page "Success" - External user can register the operation

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-registration#1905]
#### Description:
As an industry user, given that the operation has been added to my operator, I can register the operation for IRC approval.

#### Acceptance Criteria:

Given that I am on the submission page
When I click sign and register
Then I am brought to the success page 
And this operation’s status will now be Registered in my operations table.
[Figma](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=14779-436&t=qiVuWLxB21oqZtHM-4)

On the success page, 
When I click Report a Change, 
Then it will take me to the report change form

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- Blocked by #1897 
- Blocking

**Comment by shon-button** [comment]
@patriciarussellCAS

Reachable at:
https://cas-bciers-frontend-dev.apps.silver.devops.gov.bc.ca/registration/register-an-operation

**Comment by patriciarussellCAS** [comment]
Unable to review and test this workflow. Blocked until #2133 is addressed. 
Moved to current sprint.

**Comment by patriciarussellCAS** [comment]
My review notes: 

- Success page works as intended and matches wireframe
- When clicking on "Report a Change" I get a Not Found notification (see screenshot)
- Is this being worked on in another ticket? Do we have confirmation where this button should take the user or is this still being clarified with BA? 
cc: @zoeyli-46 

![Image](https://github.com/user-attachments/assets/cc8f55e2-f4b9-4507-8d88-549f9c4cf782)

**Comment by marcellmueller** [comment]
We don't have that feature implemented. However we do have a missing URL tracking PR here. I've added that so we don't forget to update the URL:

https://github.com/bcgov/cas-registration/issues/2121

**Comment by patriciarussellCAS** [comment]
Thanks Marcel! Since the URL issue is captured in 2121 I will close this ticket.
